components/Question/AskQuestionForm.tsx

```diff
@@ -73,6 +77,14 @@ function AskQuestionForm({ post, user, onExit }: AskQuestionFormProps) {
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
   const currentUser = useCurrentUser();
   const documentContext = useContext(DocumentContext);
+  const [withBounty, setWithBounty] = useState<boolean>(false);
+  const [bountyOffered, setBountyOffered] = useState<number>(0);
+  const [bountyError, setBountyError] = useState<any>(null);
+  const alert = useAlert();
+  const handleBountyInputChange = ({ hasError, errorMsg, value }) => {
+    setBountyOffered(value);
+    setBountyError(errorMsg);
```

Comment:
it doesn't seem like we show the error message anywhere, e.g. if I don't have enough balance, or am below the minimum required.

Currently if there's an error in the `BountyInput` I just see a disabled button and have to guess why it's not letting me submit.

Reply:
Thanks for letting me know. I did a rebase and I think that might have gotten messed up. There should be an error in there.